templates/templates/_navigation-enterprise.html

```diff
@@ -1,31 +1,80 @@
 <div class="row fade-animation" id="enterprise-tab-content">
+  <blockquote class="u-no-margin--top">
+    <p style="max-width: 50em">Ubuntu is the leading Linux for cloud, containers, machine learning and scale-out workloads.</p>
+  </blockquote>
   <div class="col-12 p-divider">
-    <div class="col-4 p-divider__block">
-      <h4><a class="p-link--soft" href="/containers" title="Visit the containers pages">Containers&nbsp;&rsaquo;</a></h4>
-      <p>Dramatically improve VM density with <a href="https://linuxcontainers.org/" title="Visit the LXD - external site">LXD</a> containers for traditional apps, and build new cloud-native apps with <a href="/kubernetes">Kubernetes on Ubuntu</a>, for perfect portability across Google and Azure cloud kubernetes services. Ubuntu is the top OS for <a href="/containers/docker-ubuntu">Docker deployment and ops</a>.</p>
-      <h4><a class="p-link--soft" href="/openstack" title="Visit the private cloud page">OpenStack&nbsp;&rsaquo;</a></h4>
-      <p>We offer fully managed <a href="/openstack">Openstack</a> as a cost-effective VMware replacement. Ubuntu is the number one platform for OpenStack in large-scale deployments, globally.</p>
-      <h4><a class="p-link--soft" href="/internet-of-things" title="Visit the IoT pages">Internet of Things&nbsp;&rsaquo;</a></h4>
-      <p><a href="https://snapcraft.io" title="Visit Snapcraft - external site">Snaps</a> are secure Linux packages with automatic updates for devices and desktops. <a href="/core">Ubuntu Core</a> is a high-security version of Ubuntu with full over-the-air updates for every component in the system, supported on ARM and Intel SBCs and SoCs.</p>
-      <h4><a class="p-link--soft" href="/cloud/public-cloud">Public cloud&nbsp;&rsaquo;</a></h4>
-      <p>Ubuntu is the OS most used on <a href="/cloud/public-cloud">AWS</a>, <a href="/cloud/public-cloud">Azure</a>, and <a href="/cloud/public-cloud">Google Cloud</a> and is the default OS for <a href="/cloud/public-cloud">Google Kubernetes</a> and the <a href="/cloud/public-cloud">Azure Kubernetes Service</a>. We offer <a href="/support">enterprise support for cloud users</a> of Ubuntu.</p>
-      <h4>Machine Learning</h4>
-      <p>The leading platform for machine learning and AI products, development and research. Try <a href="https://www.tensorflow.org/" title="Visit TensorFlow - external site">TensorFlow</a> from Google, and <a href="https://developer.nvidia.com/deep-learning-software" title="Visit Nvidia developer - external site">Nvidia’s ML stack</a>, or the <a href="https://aws.amazon.com/machine-learning/" title="Visit AWS Machine Learning - external site">AWS ML toolchain</a>, all on Ubuntu.</p>
+    <div class="col-4 p-divider__block navigation-col">
+
+      <h4 class="p-heading--navigation"><a class="p-link--soft" href="/containers" title="Visit the containers pages">Containers</a></h4>
+      <p class="p-p--navigation">The base OS for both Google and Azure Kubernetes.</p>
```

Comment:
Copy docs said "Ubuntu is the base OS for the Google and Azure cloud kubernetes services"